indico/modules/events/reminders/models/reminders.py

```diff
@@ -210,8 +210,8 @@ def send(self):
         email_tpl = make_reminder_email(self.event, self.include_summary, self.include_description, self.message)
         attachments = []
         if self.attach_ical:
-            event_ical = event_to_ical(self.event, skip_access_check=True)
-            attachments.append(('event.ics', event_ical, 'text/calendar'))
+            event_ical = event_to_ical(self.event, skip_access_check=True, method='REQUEST')
+            attachments.append(MIMECalendar('event.ical', event_ical))
```

Comment:
`.ical`? don't almost all ical attachments use `.ics`? (e.g. `vcalendar.ics` from outlook)

Reply:
Yeah, it's a typo.